### ISSUE TITLE
Remove mock location permission during build process

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tripodtr/npr-android-app#91]
```
1. Check the manifest file or the market entry for the app
2. Note it allows mock location to check GPS in dev mode
3. Users have written in to complain.

What is the expected output? What do you see instead?
NPR should remove this unnecessary permission by hand before production or a 
build process should be added.


```

Original issue reported on code.google.com by `jpenn...@gmail.com` on 14 Jun 2011 at 8:12

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 2.1
```

Original comment by `jpenn...@gmail.com` on 17 Jun 2011 at 7:21
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Done.
```

Original comment by `jpenn...@gmail.com` on 16 Nov 2011 at 5:10
- Changed state: **Live**
